lib/navigation/control/consts.hpp

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <optional>
+
+#include <core/types.hpp>
+#include <eigen3/Eigen/Dense>
+
+namespace hyped::navigation {
+
+constexpr std::size_t state_dimension       = 3;
+constexpr std::size_t measurement_dimension = 1;
+constexpr std::size_t extended_dimension    = 2;
+// in order acc, velocity, displacement
+using StateVector           = Eigen::Matrix<core::Float, state_dimension, 1>;
+using StateTransitionMatrix = Eigen::Matrix<core::Float, state_dimension, state_dimension>;
+using StateTransitionCovarianceMatrix
+  = Eigen::Matrix<core::Float, state_dimension, state_dimension>;
+using ErrorCovarianceMatrix = Eigen::Matrix<core::Float, state_dimension, state_dimension>;
+using MeasurementMatrix
+  = Eigen::Matrix<core::Float, measurement_dimension, state_dimension>;  //[[1, 0, 0], [0, 0, 0],
+                                                                         //[0, 0, 0]]
+using MeasurementVector = Eigen::Matrix<core::Float, measurement_dimension, 1>;  //[acc_val, 0, 0]
+using MeasurementNoiseCovarianceMatrix
+  = Eigen::Matrix<core::Float, measurement_dimension, measurement_dimension>;
+using ExtendedStateVector
+  = Eigen::Matrix<core::Float, extended_dimension, 1>;  //[Jerk, Snap, Crackle, Pop, ...]
+using JacobianMatrix
+  = Eigen::Matrix<core::Float, extended_dimension, state_dimension>;  // higher order derivatives
+                                                                      // for propogation
+
+static constexpr core::Float kTrackLength       = 100.0;  // m
+static constexpr core::Float kBrakingDistance   = 20.0;   // m TODOLater:check!
+static constexpr core::Float kPi                = 3.14159265359;
+static constexpr core::Float kWheelCicumference = kPi * 0.1;  // m TODOLater: check!
+static constexpr core::Float kStripeDistance    = 10.0;       // m TODOLater:check!
```

Comment:
if this is the keyence stripe, i think its a 6m distance from what Greg wrote in slack